recipes/ta-lib/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "TA-Lib" %}
+{% set version = "0.4.0" %}
+{% set md5 = "00e0616acfb1c77a76fb0a63f2fa7363" %}  # [win]
+{% set md5 = "308e53b9644213fc29262f36b9d3d9b9" %}  # [not win]
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://prdownloads.sourceforge.net/{{ name|lower }}/{{ name|lower }}-{{ version }}-msvc.zip  # [win]
+  url: https://prdownloads.sourceforge.net/{{ name|lower }}/{{ name|lower }}-{{ version }}-src.tar.gz  # [not win]
+  md5: {{ md5 }}
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+  run: []
+
+test:
+  commands:
+    # run ta_regtest executable
+    - ta_regtest
```

Comment:
You want to add a `conda inspect linkages {{ name }}  # [not win]` here to see that things are being built as you expect them to be.